core/controllers/profile.py

```diff
@@ -139,6 +155,26 @@ def post(self, _):
 class PreferencesHandler(base.BaseHandler):
     """Provides data for the preferences page."""
 
+
+    URL_PATH_ARGS_SCHEMAS = {}
+    HANDLER_ARGS_SCHEMAS = {
+        'GET': {},
+        'PUT': {
+            'update_type': {
+                'schema': {
+                    'type': 'list(str)'
```

Comment:
This is not a valid type, please check other places to see how we define lists in schemas.

Reply:
done @vojtechjelinek